EIPS/eip-4345.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 4345
+title: Difficulty Bomb Delay to April 2022
+description: Delays the difficulty bomb to show effect in April 2022.
```

Comment:
I think this could be slightly clearer. Maybe something like "until April 2022" or "take effect in April 2022".

Reply:
Changed, along with the actual month to better describe when the bomb will actually be noticeable.

Reply:
ty